### PR TITLE
Fixes in English translatable strings

### PR DESCRIPTION
**Pull request by dimag0g**
I came across several oddities in English text while working on translations together with @a1exsh, and I think we should fix at least some of them. The changes include:

- the use of word "day" instead of "turn" when talking about in-game days
- use "transfer"/"exchange" instead of "move" when talking about artifacts
- various small grammar / phrasing fixes

I suggest we first discuss which strings should be changed, and then I merge the new strings in the translation files and update the translation files where possible.

In case of grammar / phrasing fixes, most translations already translate the correct sentence, not the actual English sentence to be translated. Such translations only need to be re-associated with new strings. Translations which I cannot assess will get the `#fuzzy` tag, so that old translations are used for new strings, but translators will be asked to check these strings next time they work on the respective translation files.

### CODE REVIEW

**Comment by ihhub** [comment]
I'm insisting that @LeHerosInconnu should express his opinion over these changes :)

File: src/fheroes2/gui/interface_buttons.cpp [hunk 1]
```diff
@@ -193,7 +193,7 @@ fheroes2::GameMode Interface::ButtonsArea::QueueEventProcessing( void )
     else if ( le.MousePressRight( adventureRect ) )
         Dialog::Message( _( "Adventure Options" ), _( "Bring up the adventure options menu." ), Font::BIG );
     else if ( le.MousePressRight( fileRect ) )
-        Dialog::Message( _( "File Options" ), _( "Bring up the file options menu, alloving you to load menu, save etc." ), Font::BIG );
+        Dialog::Message( _( "File Options" ), _( "Bring up the file options menu, alloving you to load, save, start a new game or quit." ), Font::BIG );
```
Comment (a1exsh) [inline src/fheroes2/gui/interface_buttons.cpp:196]:
```suggestion
        Dialog::Message( _( "File Options" ), _( "Bring up the file options menu, allowing you to load, save, start a new game or quit." ), Font::BIG );
```

**Comment by a1exsh** [comment]
@dimag0g nice!

There's a merge conflict somehow already — could you have a look again?

**Comment by LeHerosInconnu** [comment]
Hello @ihhub and @dimag0g,

> I'm insisting that @LeHerosInconnu should express his opinion over these changes :)

All the changes made seem appropriate to me.
Just one thing (and then several other things), :) there is no such thing as a "Water Mill", it's a "Water Wheel".

- I suggest this:
"world: Windmill, Water Wheel and Magic Garden can be captured"
Or:
"world: Windmills, Water Wheels and Magic Gardens can be captured"
Instead of:
"world: Wind/Water Mills and Magic Gardens can be captured"

- And maybe also this change:
"world: Crystal Ball gives Identify Hero and Visions spells"
Instead of:
"world: Crystal Ball also adds Identify Hero and Visions spells"

- And this:
"world: allow to set guardian to objects"
Instead of:
"world: allow set guardian to objects"

- And this:
"heroes: allow to choose any primary skill in Arena"
Instead of:
"heroes: in Arena can choose any of primary skills"

**Comment by dimag0g** [comment]
@LeHerosInconnu I added a commit with your suggestions and a couple of additional fixes.

**Comment by LeHerosInconnu** [comment]
Hello @dimag0g,

> @LeHerosInconnu I added a commit with your suggestions and a couple of additional fixes.

Nice. :)

> Everyone, let me know if you disagree with any of the changes so far. Personally I believe we should only make changes which are clearly an improvement, and not just another equally good variant. Otherwise there is a risk of editing the same lines again and again.
> 
> I was expecting at least some disagreement, e.g. in "Monster" vs. "Creature" and "Transfer artifact" vs. "Move artifact".

I think the word "monsters" is more appropriate for the "wandering creatures" on the adventure map.
In the castles, I think it is more suited to use the word "creatures".

"Exchange" is okay with the Magic Book, as a hero can only own one Magic Book (so it would indeed be a possible exchange of Magic Book only).

Ideally:

"Transfer" is fine when transferring an artifact from one hero inventory to another hero inventory.

"Move" is suitable when changing the position of an artifact in the same inventory (but can also be used for a transfer between inventories). :)

**Comment by dimag0g** [comment]
@LeHerosInconnu I think there are two cases considered in the code (correct me if I'm wrong):

1. moving an artifact to an empty space (in the same hero's inventory or another hero's inventory). This is called "move"
2. moving an artifact to another artifact's place (again, regardless of how many heroes are involved). This is called "exchange" or "move", somewhat inconsistently

I did two changes:
- replaced "move" in cases (1) with "transfer"
- replaced "move" in cases (2) with "exchange" (as in "cannot exchange artifact")

From your comment, I understand that you would like to revert cases (1) to "move". Shall I do that while keeping "exchange" in cases (2)? Or did I misunderstand?

**Comment by Branikolog** [comment]
Hi,  @LeHerosInconnu 

> I think the word "monsters" is more appropriate 
for the "wandering creatures" on the adventure map.

I personally don't like "monsters", since we have human faction: I can't consider peasants as monsters, unless they can drink vodka without any snack...

**Comment by dimag0g** [comment]
@Branikolog Well, maybe those wandering poor fellas can do just that! Seriously though, I don't mind the word "monsters" being applied to wandering armies that your hero has to fight. Note how they magically transform into "creatures" if you have diplomacy.

Also, if you really put yourself in a Necromancer's shoes, I can see how peasants slaying your skeletons could be called "monsters". Undead lives matter!

If I'm not mistaken, I only replaced "monsters" with "creatures" in the context of hiring. I guess we all agree that "creatures" are more appropriate in this case.

**Comment by LeHerosInconnu** [comment]
Hello @dimag0g and @Branikolog,

> @LeHerosInconnu I think there are two cases considered in the code (correct me if I'm wrong):
> 
> 1. moving an artifact to an empty space (in the same hero's inventory or another hero's inventory). This is originally called "move".
> 2. moving an artifact to another artifact's place (again, regardless of how many heroes are involved). This is originally called "exchange" or "move", somewhat inconsistently.
> 
> I did two changes:
> 
> * replaced "move" in cases (1) with "transfer"
> * replaced "move" in cases (2) with "exchange" (as in "cannot exchange artifact")
> 
> From your comment, I understand that you would like to revert cases (1) to "move". Shall I do that while keeping "exchange" in cases (2)? Or did I misunderstand?

I think we should do it this way:
"Move" to displace an artifact into a free slot, and "Exchange" to displace an artifact into a slot already occupied by another artifact (swap their positions) and this regardless if it is for a single inventory or for two inventories.

> Hi, @LeHerosInconnu
> 
> > I think the word "monsters" is more appropriate
> > for the "wandering creatures" on the adventure map.
> 
> I personally don't like "monsters", since we have human faction: I can't consider peasants as monsters, unless they can drink vodka without any snack...

"Monsters" would be used for opponents, and "creatures" whenever they want to join and/or when they can be recruited. :)

> If I'm not mistaken, I only replaced "monsters" with "creatures" in the context of hiring. I guess we all agree that "creatures" are more appropriate in this case.

Yes, this is the way to go. :)

**Comment by dimag0g** [comment]
Ok, I have put the "Move %{name}" strings back, and replaced "Cannot move artifact" with a more specific "Cannot move the Spellbook": as far as I can tell, that's the actual reason for the action being impossible, as cursed artifacts are handled elsewhere. How does that sound?

If there are no other suggestions, I will proceed to merge new strings with translation files.

**Comment by dimag0g** [comment]
I suppose I will leave out the update to `ru.po`, as PR https://github.com/ihhub/fheroes2/pull/4907 is already taking too long and the last thing it needs is an extra merge :)

**Comment by ihhub** [comment]
Hi @dimag0g , are we okay to merge this pull request? I see that this is somehow in a dangling state waiting for someone's reply.

**Comment by dimag0g** [comment]
@LeHerosInconnu @a1exsh @Branikolog 
If you guys have time, I would be grateful if you could take a look on the merge result for `fr.po`, `de.po` and `ru.po`, respectively, before this is merged.

**Comment by LeHerosInconnu** [comment]
Hello @dimag0g,

> @LeHerosInconnu @a1exsh @Branikolog If you guys have time, I would be grateful if you could take a look on the merge result for `fr.po`, `de.po` and `ru.po`, respectively, before this is merged.

Here are my suggestions:

2110 msgid "Buy Monsters"
msgstr "Acheter créatures"
msgid "Buy Creatures"
msgstr "Acheter des Creatures"

- "Acheter des créatures"

2627 msgid "Defeat all enemy heroes and towns."
msgstr "Détruire tous les héros et villages ennemis."

- "Vaincre tous les héros et villages ennemis."

2633 msgid "Defeat a specific hero."
msgstr "Détruire un héros spécifique."

- "Vaincre un héros spécifique."

5077 "Niché parmi les arbres se trouve un voyant aveugle. Lorsque vous lui "
"expliquez l'intention de votre voyage, le voyant active sa boule de cristal, "
"vous permettant de voir les forces et les faiblesses de vos adversaires."

- "Niché parmi les arbres se trouve un devin aveugle. Lorsque vous lui "
"expliquez l'intention de votre voyage, le devin active sa boule de cristal, "
"vous permettant de voir les forces et les faiblesses de vos adversaires."

7554 "L'%{name} rend %{count} points de magie supplémentaires par jour à votre "

- "L'%{name} restaure %{count} points de magie supplémentaires par jour à votre "

8556 msgid "world: scouting skill shows extended content info"
msgstr "monde : reconnaissance permet d'afficher des infos supplémentaires"

- "monde : Reconnaissance affiche des infos supplémentaires"

8559 msgid "world: allow to set guardian to objects"
msgstr "monde : sort Gardien applicable aux objets divers"

- "monde : autoriser le placement de gardiens pour certains lieux"

8562 msgid "world: Eagle Eye also works like Scholar in H3."
msgstr "monde : oeil d'aigle fonctionne aussi comme Érudition dans HoMM3"

- "world: Eagle Eye also works like Scholar in Heroes 3"
"monde : Oeil d'Aigle fonctionne aussi comme Érudition dans Heroes 3"

8565 msgid "world: Crystal Ball gives Identify Hero and Visions spells"
msgstr ""
"monde : La Boule de Cristal ajoute les sorts Identifier Héros et Visions"

- "monde : la Boule de Cristal donne les sorts Identifier Héros et Visions"

8569 msgid "world: Neutral armies scale with game difficulty"
msgstr "monde : armées neutres adaptées à la difficulté du jeu"

- "monde : le nombre de créatures des armées neutres augmente avec la difficulté du jeu"

8572 msgid "world: use unique artifacts providing resources"
msgstr "monde : utiliser des artéfacts uniques pour affecter les ressources"

- "monde : prendre en compte un seul exemplaire du même artéfact pour les ressources"

8575 msgid "world: use unique artifacts affecting primary skills"
msgstr "monde : utiliser des artéfacts uniques pour les compétences primaires"

- "monde : prendre en compte un seul exemplaire du même artéfact pour les compétences primaires"

8578 msgid "world: use unique artifacts affecting secondary skills"
msgstr ""
"monde : utiliser des artéfacts uniques pour les compétences secondaires"

- "monde : prendre en compte un seul exemplaire du même artéfact pour les compétences secondaires"

8582 msgid "world: Windmills, Water Wheels and Magic Gardens can be captured"
msgstr ""
"monde : moulins/roues hydrauliques et jardins magiques peuvent être capturés"
"monde : moulins, roues hydrauliques et jardins magiques peuvent être capturés"

- "monde : les Moulins, Roues Hydrauliques et Jardins Magiques peuvent être capturés"

8586 msgid "world: disable Barrow Mounds"
msgstr "monde : désactivation des Tumulus"

- "monde : désactiver les Tumulus"

8598 msgid "heroes: allow buy a spellbook from Shrines"
msgstr "héros : possibilité d'acheter un Grimoire dans les sanctuaires"

- "héros : permettre l'achat d'un Grimoire dans les Sanctuaires"

8608 msgid "heroes: allow transcribing scrolls (needs: Eye Eagle skill)"
msgstr "héros : transcription des parchemins (avec l'Oeil d'Aigle)"

- "héros : permettre la transcription des Parchemins (avec Oeil d'Aigle)"

8611 msgid "heroes: allow to choose any primary skill in Arena"
msgstr "héros : l'arène permet de choisir parmi les compétences primaires"

- "héros : permettre le choix parmi toutes les compétences primaires dans l'Arène"

8614 msgid "battle: show army order"
msgstr "combat : afficher l'ordre de l'armée"

- "combat : afficher l'ordre de jeu des créatures"

8617 msgid "battle: allow soft wait for troops"
msgstr "combat : mise en attente des troupes"

- "combat : permettre aux créatures d'attendre au cours du combat"

8620 msgid "battle: reverse wait order (fast, average, slow)"
msgstr "combat : ordre d'attente inversé (rapide, moyen, lent)"

- "combat : inverser l'ordre de jeu des créatures qui ont attendu (rapide, moyen, lent)"

8700 "Check the newest version of the game at\n"
"https://github.com/ihhub/\n"
"fheroes2/releases"
msgstr ""
"Cherchez la dernière version du jeu sur\n"
"https://github.com/ihhub/\n"
"fheroes2/releases"

- "Téléchargez la dernière version du jeu sur\n"
"https://github.com/ihhub/\n"
"fheroes2/releases"

**Comment by dimag0g** [comment]
@LeHerosInconnu Thanks for these fixes. I a few remarks, mostly about translations which are too long to fit:

- 8570 "monde : le nombre de créatures des armées neutres augmente avec la difficulté du jeu"
"monde : la puissance des armées neutres augmente avec la difficulté du jeu" is shorter and sounds OK, don't you think?

- 8572-8580: "prendre en compte un seul exemplaire du même artéfact"
This is quite long compared to "utiliser des artéfacts uniques", are you sure it sounds so much better? I'd rather keep the existing translation.

- 8583: "monde : les Moulins, Roues Hydrauliques et Jardins Magiques peuvent être capturés"
This is too long. I removed "les" to make it shorter

- 8621: "combat : inverser l'ordre de jeu des créatures qui ont attendu (rapide, moyen, lent)"
This is too long. I removed "de jeu" to make it shorter

File: files/lang/de.po [hunk 2]
```diff
@@ -997,8 +994,8 @@ msgid ""
 "The sorceresses to the northeast are rebelling! For the good of the empire "
 "you must quash their feeble uprising on your way to the mountains."
 msgstr ""
-"Die Magierinnen im Nordosten rebellieren! Zum Wohle des Reiches müssen Sie"
-"ihren schwachen Aufstand auf Ihren Weg in die Berge niederschlagen."
+"Die Magierinnen im Nordosten rebellieren! Zum Wohle des Reiches müssen "
+"Sieihren schwachen Aufstand auf Ihren Weg in die Berge niederschlagen."
```
Comment (a1exsh) [inline files/lang/de.po:998]:
```suggestion
"Sie ihren schwachen Aufstand auf Ihren Weg in die Berge niederschlagen."
```

File: files/lang/de.po [hunk 3]
```diff
@@ -2120,14 +2120,14 @@ msgstr "Burg Optionen verlassen"
...
 
-msgid "No monsters available for purchase."
-msgstr "Keine Monster zum Kauf verfügbar."
+msgid "No creatures available for purchase."
+msgstr "Keine Kreatur zum Kauf verfügbar."
 
-msgid "Buy Monsters"
-msgstr "Monster kaufen"
+msgid "Buy Creatures"
+msgstr "Kreaturen Kaufen"
```
Comment (a1exsh) [inline files/lang/de.po:2130]:
```suggestion
msgstr "Kreaturen kaufen"
```

File: files/lang/de.po [hunk 4]
```diff
@@ -2624,6 +2624,42 @@ msgstr "The Succession Wars"
...
+msgstr "Einen bestimmten Helden verlieren."
+
+msgid "Run out of time. Fail to win by a certain point."
+msgstr "Keine Zeit mehr haben. Nicht bis zu einem bestimmten Punkt gewinnen."
+
+msgid "Loss Condition"
+msgstr "Verlustbedingungen-Icon"
+
+msgid "Defeat all enemy heroes and towns."
+msgstr "Besiegen Sie alle feindlichen Helden und Städte."
```
Comment (a1exsh) [inline files/lang/de.po:2643]: This and the following few translations are inconsistent with the loss condition translations a few lines above, let's unify them, e.g:
```suggestion
msgstr "Alle feindlichen Helden und Städte besiegen."
```

File: files/lang/de.po [hunk 5]
```diff
@@ -4699,10 +4700,10 @@ msgstr ""
 
 msgid ""
 "The lighthouse is now under your control, and all of your ships will now "
-"move further each turn."
+"move further each day."
 msgstr ""
 "Der Leuchtturm ist nun unter Ihrer Kontrolle, und alle Ihre Schiffe bewegen "
-"sich nun jede Runde weiter."
+"sich nun jeder Tag weiter."
```
Comment (a1exsh) [inline files/lang/de.po:4706]:
```suggestion
"sich nun jeden Tag weiter."
```

File: files/lang/ru.po [hunk 6]
```diff
@@ -1174,6 +1337,222 @@ msgstr "Невозможно построить: %{name}."
...
+msgstr "Бладрейн"
+
+msgid "Dragontooth"
+msgstr "Драгонтус"
+
+msgid "Greywind"
+msgstr "Грейвинд"
+
+msgid "Portsmith"
+msgstr "Портсмис"
```
Comment (a1exsh) [inline files/lang/ru.po]:
```suggestion
msgstr "Портсмит"
```

File: files/lang/ru.po [hunk 7]
```diff
@@ -1174,6 +1337,222 @@ msgstr "Невозможно построить: %{name}."
...
+msgstr "Тундара"
+
+msgid "Vulcania"
+msgstr "Вулкания"
+
+msgid "Baywatch"
+msgstr "Байвоч"
+
+msgid "Fountainhead"
+msgstr "Фонтанхейд"
```
Comment (a1exsh) [inline files/lang/ru.po]:
```suggestion
msgstr "Фонтэйнхед"
```

File: files/lang/ru.po [hunk 8]
```diff
@@ -1174,6 +1337,222 @@ msgstr "Невозможно построить: %{name}."
...
+msgstr "Вилдабар"
+
+msgid "Winterkill"
+msgstr "Винтеркил"
+
+msgid "Brindamoor"
+msgstr "Бриндамур"
+
+msgid "Lakeside"
+msgstr "Лэйксайт"
```
Comment (a1exsh) [inline files/lang/ru.po]:
```suggestion
msgstr "Лэйксайд"
```

File: files/lang/ru.po [hunk 9]
```diff
@@ -1174,6 +1337,222 @@ msgstr "Невозможно построить: %{name}."
...
+msgstr "Ньюдавн"
+
+msgid "Sorpigal"
+msgstr "Сорпигал"
+
+msgid "Avone"
+msgstr "Авон"
+
+msgid "Big Oak"
+msgstr "Биг Оак"
```
Comment (a1exsh) [inline files/lang/ru.po]:
```suggestion
msgstr "Биг-Оук"
```

File: files/lang/ru.po [hunk 10]
```diff
@@ -1174,6 +1337,222 @@ msgstr "Невозможно построить: %{name}."
...
+
+msgid "Chandler"
+msgstr "Чандлер"
+
+msgid "Erliquin"
+msgstr "Эрликиум"
```
Comment (a1exsh) [inline files/lang/ru.po]:
```suggestion
msgstr "Эрликуин"
```

File: files/lang/ru.po [hunk 11]
```diff
@@ -1174,6 +1337,222 @@ msgstr "Невозможно построить: %{name}."
...
+msgstr "Авалон"
+
+msgid "Roc Haven"
+msgstr "Рокхавен"
+
+msgid "South Mill"
+msgstr "Саусмил"
+
+msgid "Weed Patch"
+msgstr "Видпас"
```
Comment (a1exsh) [inline files/lang/ru.po]:
```suggestion
msgstr "Видпатч"
```

File: files/lang/ru.po [hunk 12]
```diff
@@ -1174,6 +1337,222 @@ msgstr "Невозможно построить: %{name}."
...
+msgstr "Хилтоп"
+
+msgid "Weddington"
+msgstr "Ведингтон"
+
+msgid "Westfork"
+msgstr "Вестфорк"
+
+msgid "Whittingham"
+msgstr "Ватингем"
```
Comment (a1exsh) [inline files/lang/ru.po]:
```suggestion
msgstr "Витингем"
```

File: files/lang/ru.po [hunk 13]
```diff
@@ -1174,6 +1337,222 @@ msgstr "Невозможно построить: %{name}."
...
+
+msgid "Cathcart"
+msgstr "Кетчкарт"
```
Comment (a1exsh) [inline files/lang/ru.po]:
```suggestion
msgstr "Кеткарт"
```

File: files/lang/ru.po [hunk 14]
```diff
@@ -1174,6 +1337,222 @@ msgstr "Невозможно построить: %{name}."
...
+
+msgid "Elk's Head"
+msgstr "Элькхеад"
```
Comment (a1exsh) [inline files/lang/ru.po]:
```suggestion
msgstr "Эльксхед"
```

File: files/lang/ru.po [hunk 15]
```diff
@@ -1174,6 +1337,222 @@ msgstr "Невозможно построить: %{name}."
...
+msgstr "Роскомон"
+
+msgid "Sherman"
+msgstr "Шерман"
+
+msgid "Yorksford"
+msgstr "Йорксфорд"
+
+msgid "Blackburn"
+msgstr "Блэкбурн"
```
Comment (a1exsh) [inline files/lang/ru.po]:
```suggestion
msgstr "Блэкбёрн"
```

File: files/lang/ru.po [hunk 16]
```diff
@@ -1174,6 +1337,222 @@ msgstr "Невозможно построить: %{name}."
...
+
+msgid "Blacksford"
+msgstr "Блэксфорд"
+
+msgid "Burton"
+msgstr "Буртон"
```
Comment (a1exsh) [inline files/lang/ru.po]:
```suggestion
msgstr "Бёртон"
```
Though, I do not insist ;)

File: files/lang/ru.po [hunk 17]
```diff
@@ -1174,6 +1337,222 @@ msgstr "Невозможно построить: %{name}."
...
+
+msgid "Pig's Eye"
+msgstr "Пигай"
```
Comment (a1exsh) [inline files/lang/ru.po]:
```suggestion
msgstr "Пигс-Aй"
```
Thinking of `Elk's Head`, maybe that one should also be spelled with a dash in Russain, but not certain.

**Comment by ihhub** [comment]
Hi @a1exsh , please focus only on changes related to text which was modified in the source code. This pull request is not for Russian translation correction.

File: files/lang/ru.po [hunk 18]
```diff
@@ -1174,6 +1337,222 @@ msgstr "Невозможно построить: %{name}."
...
+
+msgid "Viper's Nest"
+msgstr "Гнездо гадюк"
```
Comment (a1exsh) [inline files/lang/ru.po]:
```suggestion
msgstr "Деревня Гадюкино"
```
:joy: 
I see it's challenging, but seems like a weird exception...

File: files/lang/ru.po [hunk 19]
```diff
@@ -1174,6 +1337,222 @@ msgstr "Невозможно построить: %{name}."
...
+msgstr "Ломбард"
+
+msgid "Timberhill"
+msgstr "Тимберхил"
+
+msgid "Troy"
+msgstr "Троя"
+
+msgid "Forder Oaks"
+msgstr "Фордер Оак"
```
Comment (a1exsh) [inline files/lang/ru.po]:
```suggestion
msgstr "Фордер Оук"
```

File: files/lang/ru.po [hunk 20]
```diff
@@ -3047,32 +3590,42 @@ msgstr ""
...
+"shows the date."
+msgstr ""
+"Это окно сообщает информацию о состоянии вашего героя или королевства, а "
+"также показывает календарь."
 
 msgid ""
-"This window provides information on the status of your hero or kingdom, and shows the date. Left click here to cycle throungh "
-"these windows."
+"This window provides information on the status of your hero or kingdom, and "
+"shows the date. Left click here to cycle throungh these windows."
```
Comment (a1exsh) [inline files/lang/ru.po]:
```suggestion
"shows the date. Left click here to cycle through these windows."
```

**Comment by a1exsh** [comment]
> Hi @a1exsh , please focus only on changes related to text which was modified in the source code. This pull request is not for Russian translation correction.

Whoops, too late :p

**Comment by ihhub** [comment]
> > Hi @a1exsh , please focus only on changes related to text which was modified in the source code. This pull request is not for Russian translation correction.
> 
> Whoops, too late :p

You can remove your previous comments.

**Comment by dimag0g** [comment]
@a1exsh I added your fixes to `de.po`. I'd rather not try to fix Russian castle names here: I have copied them as is from Buka translation with the intention of having something, which is better than nothing. I agree some of them are pretty horrible, but let's deal with them separately.

I also took your "throungh" ->"through" fix, as well as @LeHerosInconnu's suggestion to capitalize Scouting, and re-merged all translation files. In most languages I removed the fuzzy tag in these two translation, except for Polish where I'm not sure whether Scouting should be capitalized in the Polish text as well or not.

**Comment by dimag0g** [comment]
@ihhub I think we can merge this. I assume @a1exsh took a look at `ru.po` as well, and, apart from castle names (which were previously fuzzy-translated to random words like "Gold Watch" instead of "Weed Patch" and which I now translated to sub-optimal names used in Buka version) there seems to be nothing plainly wrong with it.

**Comment by LeHerosInconnu** [comment]
Hello @dimag0g,

> @LeHerosInconnu Thanks for these fixes. I a few remarks, mostly about translations which are too long to fit:

Yes, there is also this constraint of text length. :)

> * 8570 "monde : le nombre de créatures des armées neutres augmente avec la difficulté du jeu"
>   "monde : la puissance des armées neutres augmente avec la difficulté du jeu" is shorter and sounds OK, don't you think?

Yes, it's fine. :)

> * 8572-8580: "prendre en compte un seul exemplaire du même artéfact"
>   This is quite long compared to "utiliser des artéfacts uniques", are you sure it sounds so much better? I'd rather keep the existing translation.

"prendre en compte un seul exemplaire du même artéfact" is clearer but is also longer, so let's go for "utiliser des artéfacts uniques". :)
- "monde : utiliser des artéfacts uniques pour les ressources"
- "monde : utiliser des artéfacts uniques pour les compétences primaires"
- "monde : utiliser des artéfacts uniques pour les compétences secondaires"

> * 8583: "monde : les Moulins, Roues Hydrauliques et Jardins Magiques peuvent être capturés"
>   This is too long. I removed "les" to make it shorter

Okay.

> * 8621: "combat : inverser l'ordre de jeu des créatures qui ont attendu (rapide, moyen, lent)"
>   This is too long. I removed "de jeu" to make it shorter

Okay.

**Comment by dimag0g** [comment]
@LeHerosInconnu Here you go!

**Comment by dimag0g** [comment]
@ihhub Can we perhaps merge this, so that people working on translations get the update to the latest template?

**Comment by ihhub** [comment]
@dimag0g , done! Thank you very much for your hard work!

**Pull request merged**